### DISCUSSION TITLE
YAML Configuration File Automatic Updater - Any Interest

### DISCUSSION THREAD

**Discussion by rjsears** [martomi/chiadog#292] [Ideas]
I am the primary author of ```chia_plot_manager``` (https://github.com/rjsears/chia_plot_manager) and as part of my project I have written a python script that automatically manages updating the main yaml config file when needed by adding new keys based on a skel file. 

I am going to get around to implementing chiadog and was wondering if having an automatic yaml config file updater would be helpful.

https://github.com/rjsears/chia_plot_manager/blob/main/chianas/config_file_updater.py

**Comment by martomi** [comment]
Thank you for that suggestion! Auto-upgrading the config is indeed a nice feature but I think not necessary for `chiadog` at this stage. There's no expectation to evolve the config significantly in the foreseeable future.

**Reply by rjsears** [reply]
Perfect, thank you for letting me know. Great project!
